seed/main/bho.xml

```diff
@@ -19,10 +19,10 @@ CLDR data files are interpreted according to the LDML specification (http://unic
 		</scripts>
 	</localeDisplayNames>
 	<characters>
-		<exemplarCharacters draft="contributed">[क ख ग घ डं च छ ज झ ञ ट ठ ड ढ ण ड़ ढ़ त थ द ध न प फ ब भ म य र ल व श ष स ह क्ष त्र ज्ञ श्र ा  ि  ी  ु  ू  ृ  े  ै  ो  ौ  ँ  ं  ः  ॅ  ॔  ्]</exemplarCharacters>
-		<exemplarCharacters type="auxiliary">[अ आ इ ई उ ऊ ऋ ए ऐ ओ औ अं अ:]</exemplarCharacters>
+		<exemplarCharacters draft="contributed">[क ख ग घ डं च छ ज झ ञ ट ठ ड ढ ण ड़ ढ़ त थ द ध न प फ ब भ म य र ल व श ष स ह क्ष त्र ज्ञ श्र ा  ि  ी  ु  ू  ृ  े  ै  ो  ौ  ँ  ं  ः  ॅ  ्]</exemplarCharacters>
+		<exemplarCharacters type="auxiliary">[अ आ इ ई उ ऊ ऋ ए ऐ ओ औ अ:]</exemplarCharacters>
 		<exemplarCharacters type="numbers" draft="contributed">[\- ‑ , . % ‰ + 0० 1१ 2२ 3३ 4४ 5५ 6६ 7७ 8८ 9९]</exemplarCharacters>
-		<exemplarCharacters type="punctuation" draft="provisional">[- _ – — , ; ~ : \ ! | ? . … ` ' ‘ ’ " “ ” ( ) { } [ ] § @ * / + - # ′ ″]</exemplarCharacters>
+		<exemplarCharacters type="punctuation" draft="provisional">[\- _ – — , ; ~ : \ ! | ? . … ` ' ‘ ’ " “ ” ( ) { } [ ] § @ * / + # ′ ″]</exemplarCharacters>
```

Comment:
Some of the characters in th punctuation set need to be escaped (with backslash in front of them) because they have special meaning in a UnicodeSet. In particular the square brackets, otherwise e.g. the closing one is interpreted as the end of the set. Also the colon. So these need be represented as \:   \[   \]